package/src/hooks/useIsMountedRef.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useRef } from 'react';
+
+export const useIsMountedRef = () => {
+  const isMounted = useRef<boolean | null>(null);
```

Comment:
```suggestion
  const isMounted = useRef(false);
```

It doesn't look like this is set to null anywhere, but I might be missing something?